spacy_llm/tasks/util/span.py

```diff
@@ -103,3 +114,62 @@ def parse_responses(
 
             self.assign_spans(doc, spans)
             yield doc
+
+    def to_bytes(
+        self,
+        *,
+        exclude: Tuple[str] = cast(Tuple[str], tuple()),
+    ) -> bytes:
+        """Serialize the LLMWrapper to a bytestring.
+
+        exclude (Tuple): Names of properties to exclude from serialization.
+        RETURNS (bytes): The serialized object.
+        """
+
+        def serialize_plain_config():
+            plain = {key: getattr(self, key) for key in self._PLAIN_CONFIG_KEYS}
+            return srsly.json_dumps(plain)
+
+        def serialize_examples():
+            if self._examples is None:
+                return srsly.json_dumps(None)
+            examples = [eg.dict() for eg in self._examples]
+            return srsly.json_dumps(examples)
+
+        serialize = {
+            "plain": serialize_plain_config,
```

Comment:
To make this more similar to existing methods, I'd suggest `cfg` instead of `plain`.